server/test/data/RoboCup 2021/Cocktail Party.yaml

```diff
@@ -0,0 +1,8 @@
+description:  "Enter the arena, take the orders of the three guests trying to yet your attention, serve the drinks and exit the arena"
```

Comment:
I don't like spaces in file names

Reply:
I actually do :blush: . I don't see the downsides here (not like you need to enter the path manually), you don't have to worry about replacing the spaces in the strings you're manipulating anyway and a folder name like `RoboCup_2021` and file names with Camel_Case_With_Underscores (`Cocktail_Party.yaml`) look extremely ugly ;-)